.gitmodules

```diff
@@ -1,9 +1,9 @@
 [submodule "microstrain_inertial_msgs/microstrain_inertial_msgs_common"]
 	path = microstrain_inertial_msgs/microstrain_inertial_msgs_common
-	url = https://github.com/LORD-MicroStrain/microstrain_inertial_msgs_common.git
```

Comment:
Now that the submodule PRs have been merged, the changes to this file should be reverted